### DISCUSSION TITLE
是否可以获取当前时间

### DISCUSSION THREAD

**Discussion by liuyanmei069** [timonwong/prometheus-webhook-dingtalk#154] [Q&A] [answered]
希望能实现显示持续时间，从告警触发到现在这个告警持续了多久时间。怎么在模板获取当前时间，用当前时间减去触发时间得到持续时间，这个可以怎么实现呢？

**Comment by liuyanmei069** [comment, marked as answer]
已经找到解决方式，模板里填写{{ now }}可以获取当前时间，{{ ago .StartsAt }}获取触发时间到现在过去了多久时间，2h34m7s

**Comment by xyzzen** [comment]
@liuyanmei069  老哥能告知具体写在那个文件里吗 我编辑了issue43/template.tmpl 这个文件 我看没生效 是我写的位置不对吗
